src/ui/ChatWindowFileTransferHandler.ts

```diff
@@ -3,12 +3,15 @@ import Attachment from '../Attachment'
 
 export default class FileTransferHandler {
    private handlerElement;
+   private inputElement;
 
    constructor(private chatWindow: ChatWindow) {
       this.handlerElement = this.chatWindow.getDom().find('.jsxc-file-transfer');
 
       this.handlerElement.on('click', this.showFileSelection);
 
+      this.inputElement.on('paste', this.pasteImageAsFile);
```

Comment:
Where gets the inputElement defined? At this point it should be undefined.

Reply:
you are absolutely correct. 
Fixed this with commit 19c37a4719e688ed6a17fb96b98da3ff00444543. Line 10 gets the element from the dom (using the same method as done in chatWindow).
Thanks for your patience and guidance